.github/workflows/build-and-test.yml

```diff
@@ -9,7 +9,7 @@ on:
 
 jobs:
   test:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-20.04
```

Comment:
I needed to downgrade ubuntu version from ubuntu-latest (22.04 as of the moment of writing the comment) to lower: 20.04 as ubuntu 22 is not compatible with Erlang 23 as per compatibility table here: https://github.com/erlef/setup-beam#compatibility-between-operating-system-and-erlangotp